exchange/exchange.go

```diff
@@ -629,7 +636,8 @@ func (e *exchange) getAllBids(
 	alternateBidderCodes openrtb_ext.ExtAlternateBidderCodes,
 	experiment *openrtb_ext.Experiment,
 	hookExecutor hookexecution.StageExecutor,
-	pbsRequestStartTime time.Time) (
+	pbsRequestStartTime time.Time,
+	tmaxAdjustments *config.TmaxAdjustments) (
```

Comment:
PR makes changes to pass tmaxAdjustments  as pointer within different function. This results in additional `tmaxAdjustments != nil` check whenever we need to refer any field from `tmaxAdjustments`. Not sure if we need to pass tmaxAdjustments as pointer.  

@VeronikaSolovei9 any thoughts on whether `tmaxAdjustments` should be or s not passed as pointer?

Reply:
IMO, passing a pointer is more efficient `memory wise` than passing an entire struct since that will copy the values of fields in that struct.